frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -152,6 +152,20 @@ function FileManagerMenu:setUpdateItemTable()
                 },
             }
         }
+    elseif Device.isKindle() then
```

Comment:
You don't have to add anything Kobo-specific, but my point is that you're currently locking Kindle users out of the regular screensaver functionality. Rather, you'd be adding the Kindle-specific "Enable screensaver" option to the menu so that Kindle users can then also choose to disable a specific cover for screensaver and to use a folder (if applicable). Maybe I'm being a bit unclear as I'm slightly distracted? :-)